delivery_auto_refresh/models/res_config_settings.py

```diff
@@ -7,19 +7,22 @@
 class ResConfigSettings(models.TransientModel):
     _inherit = "res.config.settings"
 
-    set_default_carrier = fields.Boolean(
```

Comment:
No need to rename these fields for making the diff bigger. I understand that is for avoiding name clashing, and I agree the initial name was not very correct, but not sure the convenience of increasing the diff. You tell me if it's only this one.